.github/workflows/build.yml

```diff
@@ -99,6 +100,11 @@ jobs:
         # this file) to a constant and skip rebuilding all of the packages each time CI runs.
         if: steps.cache_built_packages.outputs.cache-hit == ''
         run: yarn build
+      - name: Save SDK version for later
+        run: |
+          echo "Saving SDK_VERSION for later"
+          export SDK_VERSION=$(cat packages/core/src/version.ts | cut -f5 -d' ' | tr -d "'" | tr -d ";")
+          echo $SDK_VERSION > dist-serverless/version
```

Comment:
```suggestion
          cat packages/core/src/version.ts | awk -F"'" '{print $2}' > dist-serverless/version
          [ -z $(cat dist-serverless/version) ] && echo "Version extraction failed" && exit 1
```

Reply:
Nice! Thanks for that shell scripting magic!